src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -346,6 +348,10 @@ export default {
 				}
 			}
 		},
+		// Ends the search operation
+		abortSearch() {
+			this.searchText = ''
+		}
```

Comment:
> Missing trailing comma

Reply:
Shouldn't I get these in the output of make watch-js?